assignments/closure.js

```diff
@@ -1,14 +1,28 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+const sayHello = function(){
+  console.log("Hello...");
+  hellotext = "is it me you're looking for.";            
```

Comment:
make sure you tell the variable the type that you want it to be or it will default to a var which can lead to some problems down the line.